### PR TITLE
Notification System

### PR DESCRIPTION
**Pull request by AydinHassan**

### CODE REVIEW

File: module/JhHub/src/JhHub/Notification/Notification.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+class Notification implements NotificationInterface
+{
+
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @var []
```
Comment (mikeymike) [inline module/JhHub/src/JhHub/Notification/Notification.php]: lazy guy, maybe inconsistent though

Comment (AydinHassan) [inline module/JhHub/src/JhHub/Notification/Notification.php]: haha. I don't even think that's a thing :+1:

**Comment by mikeymike** [comment]
Obvs don't know the code base at all but this code looks solid :+1:

**Comment by adam-paterson** [comment]
I'm not part of the project. But I approve of this too.

**Comment by AydinHassan** [comment]
lol thanks guys!

**Pull request merged**